cli/tools/info.rs

```diff
@@ -36,10 +36,31 @@ use crate::util::checksum;
 pub async fn info(flags: Flags, info_flags: InfoFlags) -> Result<(), AnyError> {
   let factory = CliFactory::from_flags(flags).await?;
   let cli_options = factory.cli_options();
-  if let Some(specifier) = info_flags.file {
+  if let Some(mut specifier) = info_flags.file {
     let module_graph_builder = factory.module_graph_builder().await?;
     let npm_resolver = factory.npm_resolver().await?;
     let maybe_lockfile = factory.maybe_lockfile();
+    let maybe_config_file = factory.cli_options().maybe_config_file();
```

Comment:
See `factory.maybe_import_map()`. On there is a `.resolve(specifier, referrer)` function which can use `cli_options.initial_cwd()` as the referrer similar to line 64.

Reply:
Thanks @dsherret the resolve function doesn't allow `cli_options.initial_cwd()` because it expects a type of `Url`, although this works with the new changes using `imports_map.base_url()`